Source/WebCore/html/ImageInputType.cpp

```diff
@@ -136,7 +136,7 @@ void ImageInputType::attach()
     BaseButtonInputType::attach();
 
     ASSERT(element());
-    HTMLImageLoader& imageLoader = element()->ensureImageLoader();
+    HTMLImageLoader& imageLoader = protectedElement()->ensureImageLoader();
```

Comment:
It's mildly amusing that we're guarding element to call ensureImageLoader but then we're storing that to a raw reference lol. We should probably add ref() / deref() to `ImageLoader` then forward it to `m_element`. That could happen in a separate PR though.